src/index.ts

```diff
@@ -80,6 +83,28 @@ function formatInstallResults(skipFailures): string {
     .join(', ');
 }
 
+function formatDependencyInfoAnalysis(): string {
+  let output = '';
+  const {direct, shared} = dependencyInfo;
+  const formatFileInfo = files =>
+    files.map((file, index) => {
+      const {fileName, size, delta} = file;
+      let fileInfo = `${index >= files.length - 1 ? '└─' : '├─'} ${fileName} [${Math.round(
```

Comment:
nit: it would be great to move these variables out of the string declaration:

```
const lineGlyph = index >= files.length - 1 ? '└─' : '├─';
const lineSize = Math.round((size / 1000) * 100) / 100 + ' KB';
const lineDelta = // ...
return `${lineGlyph} ${fileName} [${lineSize}${lineDelta}]`;
```